docker/build_matrix.yaml

```diff
@@ -139,7 +139,7 @@
   TARGET: pytorch_stage
   TORCHVISION_VERSION: 0.19.1
   UBUNTU_VERSION: '22.04'
-- AWS_OFI_NCCL_VERSION: v1.11.0-aws
+- AWS_OFI_NCCL_VERSION: v1.14.2
```

Comment:
FYI, the aws-ofi-nccl plugin is now vended as part of the EFA installer itself starting installer v1.39 (which came with 1.14.1)... Installer v1.39 ships with aws-ofi-nccl plugin v1.14.2.

Reply:
oh nice, thanks for the heads up!